modules/connectivity/ParticipantConnectionStatus.js

```diff
@@ -347,9 +369,12 @@ export default class ParticipantConnectionStatus {
         const isVideoTrackFrozen = this.isVideoTrackFrozen(participant);
         let isConnActiveByJvb = this.connStatusFromJvb[id];
 
-        // If no status was received from the JVB it means that it's active
-        // (the bridge does not send notification unless there is a problem).
-        if (typeof isConnActiveByJvb !== 'boolean') {
+        if (this.conference.isP2PEstablished()) {
+            logger.debug('Assuming connection active by JVB - in P2P mode');
+            isConnActiveByJvb = true;
```

Comment:
i don't get this...we see that p2p is established...what does it mean to set `isConnActiveByJvb`?